fe/fe-core/src/main/java/org/apache/doris/aspectj/LogAspect.java

```diff
@@ -0,0 +1,50 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.aspectj;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.aspectj.lang.annotation.AfterThrowing;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Pointcut;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+@Aspect
+public class LogAspect {
+
+    private static final Logger LOG = LogManager.getLogger(LogAspect.class);
+
+    @Pointcut("@annotation(org.apache.doris.common.annotation.ExceptionLog)")
+    public void throwingLogPointCut() {
+    }
+
+    @AfterThrowing(value = "throwingLogPointCut()", throwing = "e")
+    public void exceptionLog(Exception e)  {
+        LOG.error(e.getMessage());
```

Comment:
Print 2 logs in one line.
And do we need to print stack trace for each exception? Maybe we can use a flag to control it?

Reply:
I added a parameter to let the user decide whether to print the stack information, the default is not to print